sky/clouds/oci.py

```diff
@@ -501,59 +511,45 @@ def _get_image_id(
         region_name: str,
         instance_type: str,
     ) -> str:
-        if image_id is None:
-            return self._get_default_image(region_name=region_name,
-                                           instance_type=instance_type)
-        if None in image_id:
-            image_id_str = image_id[None]
-        else:
-            assert region_name in image_id, image_id
-            image_id_str = image_id[region_name]
-        if image_id_str.startswith('skypilot:'):
+        image_id_str = self._get_image_tag(image_id=image_id,
+                                           instance_type=instance_type,
+                                           region=region_name)
+
+        if not image_id_str.startswith('ocid1.image'):
```

Comment:
Let's keep this `if image_id_str.startswith('skypilot:')` as that is the only case we consider it as a tag.

Reply:
ok